### ISSUE TITLE
`C-l` screws up `j`

### ISSUE THREAD

**Issue by kini** [syl20bnr/spacemacs#5772]
#### Description
If you use `C-l` to scroll the buffer until point is at the bottom of the window, pressing `j` causes point to move more than 1 line down in the buffer.

#### Reproduction guide
- Start Emacs
- Open an empty buffer
- Type `1000 i a RET ESC` to create 1000 lines containing the text "a"
- Type `500 g g` to go to the middle of the file
- Type `C-l C-l C-l` to scroll the buffer until point is at the bottom of the window
- Type `j`

*Observed behaviour:*
Point is now on line 578 (or some other number greater than 501 -- seems to depend on the size of the window you're viewing the buffer in)

*Expected behaviour:*
Point should be on line 501

#### System Info
- OS: gnu/linux
- Emacs: 24.5.1
- Spacemacs: 0.105.18
- Spacemacs branch: develop (rev. 2034195)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(spacemacs-helm auto-completion better-defaults emacs-lisp git markdown org
                (shell :variables shell-default-height 30 shell-default-position 'bottom)
                (spell-checking :variables spell-checking-enable-by-default nil)
                syntax-checking version-control rust haskell)
```

**Comment by kini** [comment]
I found that this only occurs when smooth scrolling is on, so it seems to be a bug in smooth scrolling.

**Comment by syl20bnr** [comment]
Should be fixed in develop where we removed smooth-scrolling package.

**Comment by kini** [comment]
Yup, confirmed. Thanks!

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
